projects/pgai/.python-version

```diff
@@ -1 +1 @@
-3.10
+3.12.9
```

Comment:
Why did this change?

Reply:
I'm reverting it back, I updated it for a test. 

We are using 3.12 for the docker container and I wanted the same,